[FULL CI][WIP][DO NOT MERGE] just a harmles thing to find stable ground.

jnweiger opened the pull request:
bisect, if the test setup changed wrt 10.12.0

jnweiger committed:
just a harmles thing to find stable ground.

ownclouders commented:
:boom: Acceptance tests pipeline <strong>cliDbConversion-sqlite-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/38118/115
<!-- id: pr-40698 -->

jnweiger committed:
Update Setup.php

jnweiger committed:
Update Setup.php

jnweiger committed:
spooky ci. Now it always succeeds???

jnweiger committed:
Update Setup.php

jnweiger committed:
kk

jnweiger committed:
Merge branch 'ci-test-drive-jw' of github.com:owncloud/core into ci-test-drive-jw

jnweiger committed:
we better normalize all slashes. Multiple slashes will occur in the test harness, where
we do https://github.com/owncloud/core/blob/master/tests/acceptance/run.sh#L761

Normalization is alway a good thing, when users don't if or where to include a slash in
the value of htaccess.RewriteBase

jnweiger committed:
add backslashes for coding standard

jnweiger committed:
try all ...

jnweiger committed:
all 11: killed. try with 6

jnweiger committed:
try 6 but whout syntax errors

jnweiger committed:
6 are good, try 10.

jnweiger committed:
12 out of 13!

jnweiger committed:
now, all 13...
